### ISSUE TITLE
Document Custom Elements tags being renamed

### ISSUE THREAD

**Issue by UserGalileo** [11ty/is-land#10]
Thanks for the great tool :)

I think it should be mentioned that in order to stop Custom Elements from initializing inside the island, a side-effect of this package is that their tags get renamed.

```html
<is-land on:interaction>
  <test-component></test-component>
</is-land>
```

Becomes:

```html
<is-land on:interaction>
  <is-land--test-component></is-land--test-component>
</is-land>
```

This could cause problems, styling comes to mind:

```css
/* Element without Shadow DOM */
test-component h1 { ... }
```

This wouldn't work until the island is hydrated.

**Comment by zachleat** [comment]
Yeah! I did also want to call out #14 here which was just shipped in 3.0 today

**Comment by zachleat** [comment]
I wrote up this blog post https://www.zachleat.com/web/defer-hydration/ and linked to it from the README!
